config/cloud.cfg.tmpl

```diff
@@ -252,15 +252,15 @@ system_info:
          primary: http://ports.ubuntu.com/ubuntu-ports
          security: http://ports.ubuntu.com/ubuntu-ports
    ssh_svcname: ssh
-{% elif variant in ["almalinux", "alpine", "amazon", "arch", "cloudlinux", "eurolinux",
-                    "fedora", "gentoo", "miraclelinux", "openEuler", "openmandriva", "rocky", "suse", "virtuozzo"] or is_rhel %}
+{% elif variant in ["alpine", "amazon", "arch", "fedora",
```

Comment:
By changing this, aren't we changing the default user for all of these distros? Should we instead just change line 263 to say `{% elif variant in ["rhel", "centos"] %}`?

Reply:
For line 255 and 256, I only moved the Enterprise Linux distributions from the list and tacked them on to the `is_rhel` list at the top. So that should be operating basically the same just as how it is now.

For line 263: I think you're right. Setting it to `{% elif variant in ["rhel", "centos"] %}` is probably cleaner here.

Reply:
Ah right, this line was a red herring. Thanks for the update on 263.